pkg/network/openshift_sdn_test.go

```diff
@@ -47,7 +48,7 @@ func TestRenderOpenShiftSDN(t *testing.T) {
 
 	errs := validateOpenShiftSDN(config)
 	g.Expect(errs).To(HaveLen(0))
-	FillDefaults(config)
+	FillDefaults(config, nil, net.IPv4(4, 4, 4, 4))
```

Comment:
I assume none of the tests that are calling FillDefaults() actually look at the filled-in MTU, and so they aren't going to suddenly fail if the configuration of the CI machines changes?

Reply:
Good thought, and, yes :-)